File: kotlin-spring/src/main/kotlin/org/openapitools/model/MissingArgumentError.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.AnyType
import org.openapitools.model.CodedErrorBase
import org.openapitools.model.MissingArgumentErrorAllOf
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param result 
 * @param msg 
 * @param code 
 * @param varName It contains the information about the missing parameter. 
 */
data class MissingArgumentError(

    @field:Valid
    @field:JsonProperty("result", required = true) val result: AnyType,

    @field:Valid
    @field:JsonProperty("msg", required = true) val msg: AnyType,

    @field:Valid
    @field:JsonProperty("code") val code: AnyType? = null,

    @field:JsonProperty("var_name") val varName: kotlin.String? = null
) {

}

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/MissingArgumentError.scala
package org.openapitools.server.model


/**
 * @param result  for example: ''null''
 * @param msg  for example: ''null''
 * @param code  for example: ''null''
 * @param varName It contains the information about the missing parameter.  for example: ''null''
*/
final case class MissingArgumentError (
  result: AnyType,
  msg: AnyType,
  code: Option[AnyType],
  varName: Option[String]
)

